transmartApp/grails-app/init/org/transmartproject/app/BootStrap.groovy

```diff
@@ -50,11 +50,6 @@ class BootStrap {
 
         // force marshaller registrar initialization
         grailsApplication.mainContext.getBean 'marshallerRegistrarService'
-
-        if ('clientCredentialsAuthenticationProvider' in
-                grailsApplication.config.grails.plugin.springsecurity.providerNames) {
-            OAuth2SyncService.syncOAuth2Clients()
-        }
     }
 
```

Comment:
The blocks about SAML probably also should be in transmart-server. I don't know what the purpose of the forceEagerSessionCreation and concurrent sessions stuff is, so I'm not sure if it belongs here or in -server.

There's also RScript related config that should ideally live in the R script module, but we don't need to bother with that now.

I would think the configFine check should live in tm-server (but I'm not certain). Currently it is there too, duplicated. Todo: deduplicate it. And so is the fixupConfig stuff, that probably shouldn't be in there. Does grails run the BootStrap.init's from all modules or just from the main application?

And tm-server Bootstrap.groovy and tm-oauth Bootstrap.groovy are exact copies. That also needs deduplication. I think tm-oauth doesn't need most of what's in there.